tests/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ add_subdirectory(amqp_management_ut)
 add_subdirectory(cbs_ut)
 add_subdirectory(connection_ut)
 add_subdirectory(frame_codec_ut)
+add_subdirectory(link_ut)
```

Comment:
Let's not adding here for UTs. Reason: this has to be rewritten with specs et all. The current added UTs do not have any requirements and give a false sense of safety.